### DISCUSSION TITLE
How to have CSS classes hints in vscode?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#14277] [General - Components / Directives / etc] [answered]
I read the [VSCode configuration guide](https://quasar.dev/start/vs-code-configuration) and searched for existing similar questions but did not find them - I feel that this may be a problem for me only (?)

My VSCode is correctly hinting the Quasar tags:

![image](https://user-images.githubusercontent.com/1345886/187021752-f7d9f754-5937-42cf-a867-c1f0c9f44d20.png)

It does not take into account Quasar CSS classes, though:

![image](https://user-images.githubusercontent.com/1345886/187021858-c309192b-c544-4470-aaa9-4ce451d531b0.png)

**Is there a VSCode setting specific to Quasar that I am missing?**

**Comment by yusufkandemir** [comment, marked as answer]
From our Discord server, by @ldiebold: https://discord.com/channels/415874313728688138/415880302229979137/988957182739488788

> CHECK THIS OUT!!!
> We can get css autocompletion for Quasar easily!!!
> ![img](https://cdn.discordapp.com/attachments/415880302229979137/988956724033650748/unknown.png)
>
> 1. Install the "SCSS Everywhere" VSCode Extension
> 2. Add this to your settings (you might want to do this in a `.vscode/settings.json` file so it's "per project")
> ```json
> {
>   "html-css-class-completion.remoteStyleSheets": [
>     "https://cdn.jsdelivr.net/npm/quasar@2.7.3/dist/quasar.prod.css"
>   ],
> }
> ```

https://marketplace.visualstudio.com/items?itemName=gencer.html-slim-scss-css-class-completion

@ldiebold if you have the time for it and the solution still works fine, if you can PR it that would be great.

**Comment by wsw70** [comment]
@yusufkandemir @ldiebold Thank you for the solution - it works great! 

Would you know why the CSS is loaded from a CDN when it is available from the local `node_modules` (and probably up to date with the available classes)?
